Thumbprint/Targets/Thumbprint/Components/Chips/Chip.swift

```diff
@@ -248,6 +235,11 @@ open class ChipPill: Pill {
             setNeedsLayout()
         }
     }
+    
+    public override init(adjustsFontForContentSizeCategory: Bool = true) {
+        super.init(adjustsFontForContentSizeCategory: adjustsFontForContentSizeCategory)
+        config = Config(height: 32.0, padding: 16.0, textStyle: .title8)
+    }
```

Comment:
Nit: I think we can still leave the default padding values in here as they are the overridden defaults for chip.  Then I think we should make the default initializers use `defaultHeight`, `DefaultPadding` etc.